spec/lucky/select_helpers_spec.cr

```diff
@@ -1,5 +1,7 @@
 require "../spec_helper"
 
+include ContextHelper
+
```

Comment:
This was necessary because this file was dependent on it but the problem only occurred if you ran this file individually.